frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -490,13 +490,11 @@ export const createDevfileResources = async (
         metadata: {
           ...resource.metadata,
           annotations: {
-            ...defaultAnnotations,
```

Comment:
I think we need to keep this 🤔 

If annotations are provided by the Devfile Library, we don't want to lose those. I just noticed now that I failed to follow through on Regina's work and make sure these values were in the resources created 🤦 

Can we look at merging old and new annotations and labels? Is there any concern you can think of around this?